finality-aleph/src/sync/service.rs

```diff
@@ -110,16 +108,13 @@ impl<
         ))
     }
 
-    fn backup_request(&mut self, block_id: BlockIdFor<J>) {
-        self.tasks.schedule_in(block_id, Duration::from_secs(5));
-    }
-
-    fn delayed_request(&mut self, block_id: BlockIdFor<J>) {
-        self.tasks.schedule_in(block_id, Duration::from_millis(500));
-    }
-
-    fn request(&mut self, block_id: BlockIdFor<J>) {
-        self.tasks.schedule_in(block_id, Duration::ZERO);
+    fn request_highest_justified(&mut self, block_id: BlockIdFor<J>) {
```

Comment:
I thought we decided that requesting highest justified blocks will be handled by a separate ticket in the service run loop, not by the task system - what has changed?

Reply:
I ran into some weird problems with that, but I don't remember what they were. <_<" I suspect this might be doable, so I created a task for reexamining this approach, but I wouldn't want to do it in this PR anyway, because:
1. Just adds more changes here, while it can be separated away.
2. It would make the `TaskQueue` & co. be completely unused until we actually perform block requests, which would then unnecessarily require re-adding the logic of the `TaskQueue` just after removing it in this PR.

Reply:
ok, let's leave this for another task